client/task-list/style.scss

```diff
@@ -584,3 +584,27 @@
 		border-top-right-radius: 2px;
 	}
 }
+
+.woocommerce-dashboard-card {
```

Comment:
Let's move this to `dashboard/style.scss` since it's more generically used across dashboard items.

There's also one more reference to `woocommerce-homescreen-card` that we can update to `woocommerce-dashboard-card` and then we can remove the style block for `.woocommerce-homescreen-card`.